contrib/pg_stat_statements/sql/disable_pg_stat_statements.sql

```diff
@@ -0,0 +1,5 @@
+-- start_ignore
+\! gpconfig -r shared_preload_libraries;
+\! gpconfig -r optimizer;
+\! gpstop -raq -M fast;
+-- end_ignore
```

Comment:
Please fix file ending, I guess new line symbol is missing

Reply:
Done